packages/x-components/src/components/base-dropdown.vue

```diff
@@ -76,13 +78,16 @@
 
 <script lang="ts">
   import { Identifiable } from '@empathyco/x-types';
-  import { Component, Prop, Watch } from 'vue-property-decorator';
-  import Vue from 'vue';
-  import { getTargetElement } from '../utils/html';
-  import { normalizeString } from '../utils/normalize';
-  import { isInRange } from '../utils/number';
-  import { debounce } from '../utils/debounce';
-  import { VueCSSClasses } from '../utils/types';
+  import Vue, {
+    computed,
+    defineComponent,
+    nextTick,
+    onBeforeUnmount,
+    PropType,
+    ref,
+    watch
+  } from 'vue';
+  import { getTargetElement, normalizeString, isInRange, debounceFunction } from '../utils';
```

Comment:
As we discussed on slack, maybe we should avoid importing from the barrel.

Reply:
Removed all imports from barrels for all files changed